equinix/resource_network_device_test.go

```diff
@@ -85,6 +85,80 @@ func TestNetworkDevice_createFromResourceData(t *testing.T) {
 	assert.Equal(t, expectedPrimary, primary, "Primary device matches expected result")
 }
 
+func TestNetworkDeviceWithNoInternetAccess_createFromResourceData(t *testing.T) {
```

Comment:
We don't need to create new test case to cover the new field connectivity. You can simply add one more line in the previous test case.